packages/docs/src/components/navMenu.tsx

```diff
@@ -9,56 +9,62 @@ import { Classes, IProps } from "@blueprintjs/core";
 import * as classNames from "classnames";
 import * as React from "react";
 
-import { IStyleguideSection } from "./styleguide";
+import { IHeadingNode, IPageNode, isPageNode } from "documentalist/dist/client";
```

Comment:
doesn't matter, still can't import the root in this client because the root imports the plugins which import postcss and typescript which we _do not_ want to include in the webpack bundle.